pkg/plugin/util/helpers.go

```diff
@@ -20,19 +20,22 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
 )
 
-// Deprecated: go/v4 no longer supports v1beta1 option
 // HasDifferentCRDVersion returns true if any other CRD version is tracked in the project configuration.
+// Deprecated: v1beta1 CRDs and Webhooks are not supported since k8s 1.22.
+// The features that allow users to scaffold projects using these API versions are deprecated and are no longer supported.
 func HasDifferentCRDVersion(config config.Config, crdVersion string) bool {
 	return hasDifferentAPIVersion(config.ListCRDVersions(), crdVersion)
 }
 
-// Deprecated: go/v4 no longer supports v1beta1 option
 // HasDifferentWebhookVersion returns true if any other webhook version is tracked in the project configuration.
+// Deprecated: v1beta1 CRDs and Webhooks are not supported since k8s 1.22.
+// The features that allow users to scaffold projects using these API versions are deprecated and are no longer supported.
 func HasDifferentWebhookVersion(config config.Config, webhookVersion string) bool {
 	return hasDifferentAPIVersion(config.ListWebhookVersions(), webhookVersion)
 }
 
-// Deprecated: go/v4 no longer supports v1beta1 option
+// Deprecated: v1beta1 CRDs and Webhooks are not supported since k8s 1.22.
+// The features that allow users to scaffold projects using these API versions are deprecated and are no longer supported.
```

Comment:
```suggestion
// Deprecated: v1beta1 CRDs and Webhooks are not supported since k8s 1.22.
// The features that allow users to scaffold projects using these API versions are deprecated 
// and are no longer supported.
```

Reply:
Just to make linter happy, had to do separate commit in 3.9.0: https://github.com/kubernetes-sigs/kubebuilder/pull/3271